### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by opadron**

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	url = git://github.com/opengeoscience/database_assetstore
+[submodule "submodules/girder"]
+	path = submodules/girder
+	url = git://github.com/girder/girder
+[submodule "submodules/girder_worker"]
+	path = submodules/girder_worker
+	url = git://github.com/girder/girder_worker
+[submodule "submodules/girder_ktile"]
+	path = submodules/girder_ktile
+	url = git://github.com/opengeoscience/girder_ktile
```
[the remark this answers is not in the document]
Comment (matthewma7) [reply .gitmodules]: @jbeezley I think the large_image PR doesn't introduce new features, and Minerva is not the long-term solution and also I think changing to large_image may involve some other related changes, maybe we can keep it as it is for now? If there are new features we want to include, we create separate PR for it?

[the remark this answers is not in the document]
Comment (jbeezley) [reply .gitmodules]: Sure, that's fine with me.  I just wanted to make note of the issue.

[the remark this answers is not in the document]
Comment (matthewma7) [reply .gitmodules]: 👌 Thanks for the reminder.

[the remark this answers is not in the document]
Comment (aashish24) [reply .gitmodules]: @opadron actually we would like large_image regardless of minerva. I am okay if you would like to do that as part of the next PR.

[the remark this answers is not in the document]
Comment (opadron) [reply .gitmodules]: So, I'm hearing "switch to current HEAD of minerva master" and "replace girder_ktile with current HEAD of large_image", does that sound right, @jbeezley?

[the remark this answers is not in the document]
Comment (opadron) [reply .gitmodules]: Ha! I should've hit "refresh" more.  I went ahead and tried adding large_image in.  If things look good, I'll push the commits and let you folks have a look.

File: devops/docker/docker-compose.yml [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+      "--girder-scheme", "http",
+      "--admin-name", "${GIRDER_ADMIN_NAME}",
+      "--admin-pass", "${GIRDER_ADMIN_PASS}",
+      "--user-name", "${GIRDER_USER_NAME}",
+      "--user-pass", "${GIRDER_USER_PASS}",
+      "--assetstore", "/local-assetstore",
+      "--database", "mongodb://database:27017/girder"]
+    build:
+      context: ../..
+      dockerfile: devops/docker/Dockerfile
```
[the remark this answers is not in the document]
Comment (opadron) [reply devops/docker/docker-compose.yml]: @matthewma7 you are right!  I've noticed this error and fixed it locally.  I'll post the fix, soon.

**Comment by opadron** [comment]
Thanks for the feedback, folks.  I've gotten `large_image` and the latest `minerva` working, but I'm running into a curious quirk that I wonder if you guys have seen, before?

It seems like both `large_image` and `minerva` use matplotlib and try to set its backend.  The actual backend set then becomes a function of which plugin gets loaded first.  In either case, there's a big ugly warning with traceback printed every time girder starts up.  [Here's](https://gist.github.com/opadron/be07ccf97e463008b63d82d1307b6d8f) an example.

Any ideas?

**Comment by matthewma7** [comment]
@opadron I have seen the top two warnings but the big one. @dorukozturk @manthey Have you seen the error in @opadron link?

**Comment by matthewma7** [comment]
Except for the error message. Everything else is working as expected for me. 👍 @opadron

**Comment by jbeezley** [comment]
The error should be harmless, but we should fix it to avoid confusion.  We can change [this line](https://github.com/Kitware/minerva/blob/master/server/utility/legend.py#L2) in minerva to use `TkAgg`.  If there is still a warning after doing that, we can add code in large_image and minerva to avoid running that line if the matplotlib renderer has already been initialized.

**Comment by aashish24** [comment]
thanks @jbeezley. @matthewma7 are we still using legend.py? I believe we no longer use it since now we have legend generated in GeoJS. Can we remove it if that is the case?

**Comment by matthewma7** [comment]
@aashish24 I personally think it's not being used. I'll go through a new code clean up in Minerva.

**Comment by opadron** [comment]
If @matthewma7 can update `minerva` so that we avoid the double `use()` issue, then I can update the submodule and rebuild.

**Comment by matthewma7** [comment]
@opadron The fix suggested by @jbeezley worked. I have merged the change to Minerva master. I think you could update the submodule now.

**Pull request merged**